### ISSUE TITLE
Wind direction in Planetary Conditions popup is backwards

### ISSUE THREAD

**Issue by SuperStucco** [MegaMek/megamek#4300]
Loving the conditions monitor/popup in 49.12.  However, the wind direction indicator is opposite from what it should be:

![Reversed wind direction](https://user-images.githubusercontent.com/70451747/230786464-f02352c2-3f1d-43df-a9d1-11bb4f8c072f.png)

When the wind direction is Northeast it means "from Southwest to Northeast", not "from the Northeast".  The arrow in the popup is showing the latter when it should be showing the former.

**Comment by kuronekochomusuke** [comment]
are the rules different for the normal definition of wind directions?

https://en.wikipedia.org/wiki/Wind_direction

**Comment by HammerGS** [comment]
Not that I'm aware off

**Comment by SuperStucco** [comment]
Wind is Northeast, smoke is moving to the northeast of the burning building, arrow points in the other direction:

![Smoke and wind direction](https://user-images.githubusercontent.com/70451747/230788776-5de88a05-18f4-421e-a0fc-922066f6130e.png)

Might be a question for the official rules board, then use that to resolve the conflicting information.

**Comment by HammerGS** [comment]
I think the arrow is the wrong direction (or smoke is implemented backwards).

Indicator
![image](https://user-images.githubusercontent.com/13752452/230789478-0248b627-b34a-43ca-8f60-ba02b57886ca.png)


![image](https://user-images.githubusercontent.com/13752452/230789426-0bf30c95-ef3a-4a16-bd8f-d20548ab9fb8.png)

one turn later with moderate gale

![image](https://user-images.githubusercontent.com/13752452/230789447-e2409652-2bb0-4852-8620-fa79a91dd7cb.png)

**Comment by kuronekochomusuke** [comment]
the arrow follows the standard definition for wind direction

**Comment by HammerGS** [comment]
Yeah, I think the issue is we have smoke blowing again the arrow direction and I think it will create some confusion.

**Comment by gsparks3** [comment]
It looks like the issue is that the BT wind rules don't actually define names for the wind directions, therefore we cannot tell which convention was used. Per TO:AR page 55, you roll to determine a hexside for the wind direction, but it never uses names like "northeast" or "south". The fire spread and smoke drift rules reference either "downwind" or "in the direction of the prevailing wind". To me this _implies_ (but does not clearly state) that fire should spread or smoke should drift towards the hexside indicated by the 1d6 roll. By the IRL convention for wind directions that implies that the _name_ for the wind direction would actually be based on the opposite hexside (e.g. in the example initially submitted, smoke is drifting to the northeast and the appropriate name by IRL wind convention would be "southwest"). It might be simplest / least confusing to simply remove the names and use only the arrow ("smoke drifts this way / fire spreads this way"), since that's the only thing it actually affects.
